### PR TITLE
Host Server Fix

### PR DESCRIPTION
**Pull request by richardzeng1**
-Changed php directory for host server
-Set loading to false when error

### CODE REVIEW

File: js/component-sections.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ window.Sections = Vue.component("Sections", {
                 this.section_data = data;
                 this.loading = false;
             })
-            .catch(err => (this.error = err.toString()));
+            .catch(err => {this.error = err.toString(), loading=false});
```
Comment (siefkenj) [inline js/component-sections.js]: Don't use the comma operator...ever...

File: js/component-sections.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ window.Sections = Vue.component("Sections", {
                 this.section_data = data;
                 this.loading = false;
             })
-            .catch(err => {this.error = err.toString(), loading=false});
+            .catch(err => {this.error = err.toString(); loading=false});
```
Comment (siefkenj) [inline js/component-sections.js]: Also, shouldn't it be `this.loading`. Also, prettier.

File: index.html [hunk 3]
```diff
@@ -19,9 +19,9 @@
             <div id="page">
                 <navbar>
                     <h1>Tutorial Feedback</h1>
-                    <router-link to="/course">Go to Classes</router-link>
+                    <!-- <router-link to="/">Go to Classes</router-link>
```
Comment (siefkenj) [inline index.html]: Delete, don't comment.

File: js/app.js [hunk 4]
```diff
@@ -10,7 +10,7 @@
 // `Vue.extend()`, or just a component options object.
 // We'll talk about nested routes later.
 const routes = [
-    { path: "/course", component: Courses },
+    { path: "/", component: Courses },
```
Comment (siefkenj) [inline js/app.js:13]: What happens if someone goes to `/course`? What route does it default to? There should be a way to set a default route, perhaps.

Comment (richardzeng1) [reply js/app.js:13]: I can create a 404 error component

**Pull request merged**